infra/base-images/base-builder/compile

```diff
@@ -24,7 +24,7 @@ fi
 
 if [[ $ARCHITECTURE == "i386" ]]; then
     export CFLAGS="-m32 $CFLAGS"
-    export CXXFLAGS_EXTRA="-L/usr/i386/lib $CXXFLAGS_EXTRA"
+    cp -R /usr/i386/lib/* /usr/lib
```

Comment:
Why not just change it in -DCMAKE_INSTALL_PREFIX to point to /usr ?
infra/base-images/base-clang/checkout_build_install_llvm.sh:      -DCMAKE_INSTALL_PREFIX=/usr/i386/

Reply:
>Why not just change it in -DCMAKE_INSTALL_PREFIX to point to /usr ?

I think this will break compilation of amd64 binaries. Same reason we don't do that for MSAN libs.

Reply:
Ok sounds good, lets keep as-is.

Reply:
Actually it's not clear to me this will break builds since the x64 libc++ libs are elsewhere. I'll look into this.